roles/restore/tasks/postgres.yml

```diff
@@ -1,5 +1,9 @@
 ---
 
+- name: Set Postgres Configuration Secret name
+  set_fact:
+    postgres_configuration_secret: "{{ spec['postgres_configuration_secret'] | default(postgres_configuration_secret) }}"
+
```

Comment:
This default was getting used instead of the one set in the awx spec - https://github.com/ansible/awx-operator/blob/devel/roles/restore/vars/main.yml#L10-L14

I only left those defaults in in case users specified `backup_dir` and `backup_pvc` instead of just the `backup_name` for the CR.

Solution, we get the postgres-configuration-secret-name from the backup's awx_object (spec).  A valid backup will always have the pg secret name in the spec.  

```
TASK [restore : debug] ***********************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "dev-restore-postgres-configuration"
}
TASK [restore : Set Postgres Configuration Secret name] **************************************************************************************************************************************************************************************
ok: [localhost]
TASK [restore : debug] ***********************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "awx-postgres-configuration"
}
```